<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutDialog</class>
 <widget class="QDialog" name="AboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About Anno</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Annotation Tool</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="copyright_label">
     <property name="text">
      <string>Copyright</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="version_label">
     <property name="text">
      <string>Version</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="opencv_version_label">
     <property name="text">
      <string>OpenCV Version</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="build_time_label">
     <property name="text">
      <string>Build Time</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="svn_revision_label">
     <property name="text">
      <string>SVN Revision</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="svn_path_label">
     <property name="text">
      <string>SVN Path</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>http://www.urobots.io</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
